browser-test/src/currency.test.ts

```diff
@@ -1,4 +1,4 @@
-import {AdminPrograms, AdminQuestions, ApplicantQuestions, loginAsAdmin, loginAsTestUser, logout, selectApplicantLanguage, startSession, resetSession} from './support'
```

Comment:
I don't know why this file is showing up, it's from a committed PR.

Reply:
Hmmm yeah that's weird! Did you merge in main?

Reply:
I'm pretty sure I did... It seems to all work out based on previous instance of this though...